src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -9069,7 +9073,8 @@ static bool isCandidateUnique(ref MethodSymbol? method, MethodSymbol candidate)
                         parameterTypes[i],
                         parameterRefKinds.IsDefault ? RefKind.None : parameterRefKinds[i],
                         parameterScopes.IsDefault ? DeclarationScope.Unscoped : parameterScopes[i],
-                        parameterDefaultValues.IsDefault ? null : parameterDefaultValues[i])
+                        parameterDefaultValues.IsDefault ? null : parameterDefaultValues[i],
+                        isParams: hasParamsArray && i == parameterTypes.Length - 1)
```

Comment:
I think it's worth asserting that `parameterTypes[i].IsSzArray()` for the last parameter when we have a params array.